auth4genai/mcp/guides/registering-your-mcp-client-application.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Registering your MCP Client Application
+description: Register MCP clients and servers so AI Agents can safely connect to your resources using OAuth 2.1.
```

Comment:
Did we want to say "Register [...] servers" here? 

I might say "AI Agents and tools" as well (e.g. MCP Inspector isn't an Agent)